workspaces/sandbox/plugins/sandbox/src/api/AnsibleBackendClient.tsx

```diff
@@ -47,9 +49,13 @@ export class AAPBackendClient implements AAPService {
   getAAP = async (namespace: string): Promise<AAPData | undefined> => {
     const kubeApi = await this.kubeAPI();
     const projectAAPUrl = `/apis/aap.ansible.com/v1alpha1/namespaces/${namespace}/ansibleautomationplatforms`;
-    const response = await this.fetchApi.fetch(`${kubeApi}${projectAAPUrl}`, {
-      method: 'GET',
-    });
+    const response = await fetchWithAuth(
+      this.oauthApi,
```

Comment:
My thought was that this fetch would be similar to fetchApi where you don't need to pass down the oauthApi explicitly in every request.